### PR TITLE
[EventHubs] Final README update

### PR DESCRIPTION
**Pull request by yunhaoling**
* Readme final update

* Update links

* update async sample in readme

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhubs/README.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ from azure.eventhub import EventData
 try:
     connection_str = '<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>'
     event_hub_path = '<< NAME OF THE EVENT HUB >>'
-    client = EventHubProducerClient.from_connection_string(connection_str, event_hub_path)
+    client = EventHubProducerClient.from_connection_string(connection_str, event_hub_path=event_hub_path)
 
     event_data_batch = await client.create_batch(max_size=10000)
     can_add = True
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: All of these snippets have empty error handling:
```
except:
	raise
finally:
    pass
```

We should either show actual error handling, or just remove these altogether.

File: sdk/eventhub/azure-eventhubs/README.md [hunk 2]
```diff
@@ -393,10 +407,10 @@ For instance, this error is raised if you try to send an EventData that is alrea
...
-- [./samples/async_examples/recv_async.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs/examples/async_examples/recv_async.py) - async/await support of a EventHubConsumerClient
+- [./samples/sync_samples/send.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs/samples/sync_samples/send.py) - use EventHubProducerClient to publish events
+- [./samples/sync_samples/recv.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs/samples/sync_samples/recv.py) - use EventHubConsumerClient to consume events
+- [./samples/async_samples/send_async.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs/samples/async_samples/send_async.py) - async/await support of a EventHubProducerClient
+- [./samples/async_samples/recv_async.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhubs/samples/async_samples/recv_async.py) - async/await support of a EventHubConsumerClient
 
 ### Documentation
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: We should hide this URL - something like:
```
Reference documentation is available [here](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b5/azure.eventhub.html)
```

File: sdk/eventhub/azure-eventhubs/README.md (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/README.md]: remove all the try, except and finally.
and make all these samples can run directly.

File: sdk/eventhub/azure-eventhubs/README.md (hunk 2 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/README.md]: hided

**Pull request merged**